src/types/index.ts

```diff
@@ -75,9 +75,9 @@ export interface Pool {
   isIsolated: boolean;
   assets: Asset[];
   // User-specific props
-  userSupplyBalanceCents?: number;
-  userBorrowBalanceCents?: number;
-  userBorrowLimitCents?: number;
+  userSupplyBalanceCents?: BigNumber;
+  userBorrowBalanceCents?: BigNumber;
+  userBorrowLimitCents?: BigNumber;
 }
```

Comment:
Using fractional cents means we need to convert all properties expressed in cents to `BigNumber` values. The ones I can see missing that update are: `liquidityCents`, `supplyBalanceCents`, `borrowBalanceCents`